app/controllers/package_controller.rb

```diff
@@ -70,7 +70,8 @@ def category
     categories = (mapping.blank? ? [@category] : mapping.first[:categories])
 
     app_pkgs = @appdata[:apps].reject { |app| (app[:categories].map(&:downcase) & categories.map(&:downcase)).blank? }
-    @packagenames = app_pkgs.map { |p| p[:pkgname] }.uniq.sort_by { |x| @appdata[:apps].select { |a| a[:pkgname] == x }.first[:name] }
```

Comment:
Btw, this code needs to be refactored :see_no_evil: but let's do that in another moment... :sleeping: